packages/pwa/webpack.config.js

```diff
@@ -1,9 +1,23 @@
-/*
- * Copyright (c) 2021, salesforce.com, inc.
- * All rights reserved.
- * SPDX-License-Identifier: BSD-3-Clause
- * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
- */
-const config = require('pwa-kit-build/configs/webpack/config')
-
-module.exports = config
+const webpackConfig = require('pwa-kit-build/configs/webpack/config')
+
+const fs = require ('fs')
+const path = require('path')
+
+const projectDir = process.cwd()
+const sdkDir = path.resolve(path.join(__dirname, '..', '..', '..'))
```

Comment:
This doesn't look to point to a path in the monorepo! This code won't work in a real project.

Reply:
Minor, but `path.resolve(path.join(...))` is redundant. [`path.resolve`](https://nodejs.org/docs/latest-v14.x/api/path.html#path_path_resolve_paths) (with one arg) converts a relative path to an absolute one, but [`__dirname`](https://nodejs.org/docs/latest-v14.x/api/modules.html#modules_dirname) is already absolute.